meetings/2022-03-02.md

```diff
@@ -0,0 +1,119 @@
+# W3C Solid Community Group: Authorization Panel
+
+* Date: 2022-03-02T14:00:00Z
+* Call: https://meet.jit.si/solid-authorization
+* Chat: https://gitter.im/solid/authorization-panel
+* Repository: https://github.com/solid/authorization-panel
+
+
+## Present
+* Wouter Termont
+* elf Pavlik
+* Matthieu Bosquet
+* Justin Bingham
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Wouter
+* Elf
+* Matthieu
+
+
+---
+
+## Topics
+
+* Default client based restrictions
+* [Define responsibilities of Authorization Server #250](https://github.com/solid/data-interoperability-panel/issues/250)
+
+
+### Default client based restrictions
+
+URL: https://solid.github.io/authorization-panel/acp-specification/#client-matcher
+URL: https://github.com/solid/authorization-panel/issues/80
+
+* Elf: ACP mentions the ACP Public Client which matches all clients. It's a way to explicitly say that any client is allowed. What is the difference between any Public Client and not mentioning it at all.
+* Matthieu: Yes, there is not very much of a difference. If you did not mention any client than it would be the same. If you only match on the agent, lets say Bob, and there is not policy denying otherwise than bob has access mentioned.
```

Comment:
```suggestion
* Matthieu: Yes, there is not very much of a difference. If you did not mention any client then it would be the same. If you only match on the agent, lets say Bob, and there is no policy denying otherwise, then bob has access mentioned.
```